### PR TITLE
Compelte feature mark favorite to the field type

### PR DESCRIPTION
**Pull request by nguyentrieutien1**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/73662)

## WHAT (optional)
- Create feature mark favorite to the field type
## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Comment by nguyentrieutien1** [comment]
ready

File: app/controllers/fields_controller.rb [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
 class FieldsController < ApplicationController
-  def show
+  before_action :load_field, only: :show
+
+  def show; end
+
+  private
+
+  def load_field
     @field = Field.find params[:id]
```
Comment (binhpt-1239) [inline app/controllers/fields_controller.rb:11]: Dùng find_by

File: app/views/favorite_field_types/create.turbo_stream.erb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<%= turbo_stream.replace @dom_id do %>
+  <%= render partial: "field_types/field_type", locals: { field_type: @favorite.field_type } %>
+<% end %>
+
```
Comment (binhpt-1239) [inline app/views/favorite_field_types/create.turbo_stream.erb:4]: Dư line

**Review by binhpt-1239 — COMMENTED** [review]
Fix conflic

**Pull request closed without merging**